### PR TITLE
Change get_started to recommened Gz version per platform for installations

### PR DESCRIPTION
**Pull request by j-rivero**
# 🎉 New feature

## Summary

The PR tries to simplify the election of the Gz version for the users. Basically it provides a recommendation in the landing page of main docs page https://gazebosim.org/docs and the landing page of releases (i.e https://gazebosim.org/docs/garden). It also includes a clear warning about Windows and leave it out of the table.

The PR will require some effort when we release a new version or when a version is EOL but I think that could worth it.

### Current situation problems

I think that there is a number of issues currently that are a source of confusion:

 * The Gz version selected in the top left corner named "Release" is not preserved when clicking links. 
 * The Gz version selected by default in the top left corner named "Release" is the one in development which leaves the user without binary install methods.

Both points in combination cause the effect of:
When being in a landing page of a given distribution (i.e Garden https://gazebosim.org/docs/garden), there are two links in the first step 1 install point named `latest installation tutorial here` and `Please visit the main documentation pages for a list of all releases` both are redirecting to Harmonic (development version).

### Follow up work

A new PR to be sent to clarify the options for the ROS users in the same way that [we have for gazebo-classic](https://classic.gazebosim.org/tutorials?tut=ros_wrapper_versions&cat=connect_ros).

## Checklist
- [x] Signed all commits for DCO

### CODE REVIEW

File: get_started.md [hunk 1]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
...
+|---|---|
+| Ubuntu 22.04 Jammy | [Gz Garden](docs/garden/install_ubuntu) (recommended) and [Gz Fortress](docs/fortress/install_ubuntu)
+| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_buuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
+| Ubuntu 18.04 Bionic | [Gz Citadel](docs/citadel/install_ubuntu)
+| Mac Monterey | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac BigSur | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac Catalina | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+
+Windows support via conda-forge is not fully functional, there are known runtime problems
+[stoping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).
```
Comment (mabelzhang) [inline get_started.md:23]:
```suggestion
[stopping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).
```

File: get_started.md [hunk 2]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
...
+
+If the desired plaform is not listed above or if a particular feature in a
+given Gz release is needed, there is a installation package per release
```
Comment (mabelzhang) [inline get_started.md:26]:
```suggestion
If the desired platform is not listed above or if a particular feature in a
given Gz release is needed, there is an installation package per release
```

File: get_started.md [hunk 3]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
...
-read [the latest installation tutorial here](/docs/latest/install). Please
-visit the [main documentation](/docs) pages for a list of all releases,
-along with links to their respective installation tutorials.
+The recommended installation for non expert users is the use of binary
+packages available for the platform to use:
+
+|Platform|Gz Versions|
+|---|---|
+| Ubuntu 22.04 Jammy | [Gz Garden](docs/garden/install_ubuntu) (recommended) and [Gz Fortress](docs/fortress/install_ubuntu)
+| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_buuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
```
Comment (mabelzhang) [inline get_started.md:16]:
```suggestion
| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_ubuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
```

**Review by mabelzhang — COMMENTED** [review]
Cosmetic corrections.

You know the corresponding versions better than me, so I'll trust you on that :sweat_smile: Unless someone else wants to check that.

**Comment by j-rivero** [comment]
> Cosmetic corrections.

Thanks so much!

**Review by mabelzhang — APPROVED** [review]

**Comment by azeey** [comment]
> The Gz version selected by default in the top left corner named "Release" is the one in development which leaves the user without binary install methods.

@nkoenig is looking into making the current stable version the default. If that's not visible, we can remove `Harmonic` from https://github.com/gazebosim/docs/blob/master/index.yaml until we open the tutorial party.

File: get_started.md [hunk 4]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
...
+|---|---|
+| Ubuntu 22.04 Jammy | [Gz Garden](docs/garden/install_ubuntu) (recommended) and [Gz Fortress](docs/fortress/install_ubuntu)
+| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_ubuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
+| Ubuntu 18.04 Bionic | [Gz Citadel](docs/citadel/install_ubuntu)
+| Mac Monterey | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac BigSur | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac Catalina | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+
+Windows support via conda-forge is not fully functional, there are known runtime problems
+[stopping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).
```
Comment (kscottz) [inline get_started.md:23]:
```suggestion
[Gz for Windows Issue](https://github.com/gazebosim/gz-sim/issues/168).
```

File: get_started.md [hunk 5]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
 
 ## Step 1: Install
 
-Each release of Gazebo ships with a new installation tutorial. You can
-read [the latest installation tutorial here](/docs/latest/install). Please
-visit the [main documentation](/docs) pages for a list of all releases,
-along with links to their respective installation tutorials.
+The recommended installation for non expert users is the use of binary
```
Comment (kscottz) [inline get_started.md:10]:
```suggestion
The recommended installation for new users is the use of binary
```

File: get_started.md [hunk 6]
```diff
@@ -7,10 +7,28 @@ simulation using Gazebo.
...
+| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_ubuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
+| Ubuntu 18.04 Bionic | [Gz Citadel](docs/citadel/install_ubuntu)
+| Mac Monterey | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac BigSur | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+| Mac Catalina | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
+
+Windows support via conda-forge is not fully functional, there are known runtime problems
+[stopping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).
+
+If the desired platform is not listed above or if a particular feature in a
```
Comment (kscottz) [inline get_started.md:25]:
```suggestion
If the desired platform is not listed above, or if a particular feature in a
```

**Review by kscottz — APPROVED** [review]
Better than what we have by a long shot! I approve given a few small tweaks.

**Review by azeey — APPROVED** [review]
Looks great! My only comment is that we replace all instances of `Gz` with `Gazebo`. I think adding another name (Gz) would only add to the confusion.

**Comment by j-rivero** [comment]
> Looks great! My only comment is that we replace all instances of `Gz` with `Gazebo`. I think adding another name (Gz) would only add to the confusion.

Good point. Changed in [851eac8](https://github.com/gazebosim/docs/pull/338/commits/851eac8969b6d386263f976e2abfe5c5e58a1641)

I missed a quick reference to the EOL dates for the distributions and added that in [a7b9883](https://github.com/gazebosim/docs/pull/338/commits/a7b9883ae0c2f67782604f2d6cbd95f1611ab982). Sorry for the inconvenience, @kscottz @mabelzhang @azeey if you can give it a quick look at that commit. Would be great.

**Review by azeey — APPROVED** [review]

**Comment by azeey** [comment]
@j-rivero if we can merge this in the next hour, I'll go ahead and deploy it with #341 .

**Pull request merged**